recipes/festim/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "festim" %}
+{% set version = "1.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/festim-{{ version }}.tar.gz
+  sha256: 7ebefae709a8981183cf7d118dceec71681464f496b74837a8e5a966b052d63c
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  skip: true  # [win]
```

Comment:
It is OK for noarch package to fails on Windows with a missing dependency. We prefer noarch b/c it is faster to build and the package will be ready when the missing dependency is available on Windows.